wiki/Article_styling_criteria/Writing/fr.md

```diff
@@ -282,66 +277,60 @@ Les abréviations, les acronymes et les sigles doivent prendre une majuscule, sa
 ```
 
 ## Perspective
 
-Les termes `joueur`, `utilisateur`, `skinner`, `scénariste` et `créateur` doivent être utilisés pour désigner le lecteur ou une autre personne. `ils`, `les`, `leur`, et `leurs` peuvent être utilisés si nécessaire.
+Les termes `joueur`, `utilisateur`, `skinneur`, `storyboardeur` et `créateur` doivent être utilisés pour désigner le lecteur ou une autre personne. `ils`, `eux`, `leur`, et `leurs` peuvent être utilisés si nécessaire.
 
-Les expressions `tu` et `ton` sont à éviter. Le `je` ne doit pas être utilisé. Les termes `nous`, `il` ou `sa` et `elle` ou `son` ne doivent pas être utilisés (voir ci-dessus pour les autres termes).
+Les expressions `tu` et `ton` sont à éviter. Le `je` ne doit pas être utilisé. Les termes `nous`, `il` ou `son` et `elle` ou `sa` ne doivent pas être utilisés pour désigner une personne (voir ci-dessus pour les autres termes).
 
 ---
 
-[Les articles avec registres informels](#registre-des-langues) peut ignorer cette section ; cependant, il faut éviter de mettre des `I` dans les paragraphes.
-
-## Grammar and syntax
+[Les articles avec registres informels](#registre-des-langues) peuvent ignorer cette section ; cependant, il faut éviter de mettre des `Je` dans les paragraphes.
 
-Les articles doivent utiliser de préférence une grammaire et une [syntaxe américaines plus simple](https://www.thepunctuationguide.com/british-versus-american-style.html).
+## Grammaire et syntaxe
 
-### Contractions
+Les articles doivent utiliser de préférence une grammaire et une syntaxe françaises simple.
 
-Les contractions ne doivent pas être utilisées.
+### Nombres
 
-[Les articles avec registres informels](#registre-des-langues) peuvent utiliser des contractions ; toutefois, l'usage doit rester cohérent tout au long de l'article.
-
-### Numbers
-
-*Pour le formatage des nombres, voir [Manuel de style de Wikipédia sur le formatage des nombres](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style#Numbers)*
+*Pour le formatage des nombres, voir les [conventions typographiques de Wikipédia concernant les nombres](https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Conventions_typographiques#Nombres)*
 
 [Les articles avec registres informels](#registre-des-langues) peuvent ignorer cette section ; toutefois, l'usage doit rester cohérent tout au long de l'article.
 
-### Semicolons
+### Points-virgules
 
 [Les articles avec registres informels](#registre-des-langues) doivent éviter d'utiliser les points-virgules (`;`).
 
 ### Virgule en série
 
 La virgule de série, également connue sous le nom de virgule d'Oxford ou de Harvard, doit être utilisée.
 
-### Em dash
+### Citations
+
+Consultez les [conventions typographiques de Wikipédia concernant les citations](https://fr.wikipedia.org/wiki/Wikip%C3%A9dia:Citation).
+
+Les guillemets ne se ferment après la ponctuation que si celle-ci ne peut pas être séparée de la citation (si cette citation forme une phrase complète, par exemple).
```

Comment:
you still need to indicate that the information was actually taken from the article, for example "The page mentions that..." or "According to said Wikipedia article, ..."